### PR TITLE
add crnn demo

### PR DESCRIPTION
**Pull request by Charles-258**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,45 +3,50 @@
...
 
 Guidelines:
+
 - Clone this repo to download all models and demo scripts:
-    ```shell
-    # Install git-lfs from https://git-lfs.github.com/
-    git clone https://github.com/opencv/opencv_zoo && cd opencv_zoo
-    git lfs install
-    git lfs pull
-    ```
```
Comment (fengyuentau) [inline README.md:12]: One PR for one purpose. Please create another PR for this change.

File: README.md [hunk 2]
```diff
@@ -3,45 +3,50 @@
...
+| [MobileNet-V1](./models/image_classification_mobilenet) | 224x224    | 9.04           | 92.25        | 33.18           | 145.66\*     | ---         |
+| [MobileNet-V2](./models/image_classification_mobilenet) | 224x224    | 8.86           | 74.03        | 31.92           | 146.31\*     | ---         |
+| [PP-HumanSeg](./models/human_segmentation_pphumanseg)   | 192x192    | 19.92          | 105.32       | 67.97           | 74.77        | ---         |
+| [WeChatQRCode](./models/qrcode_wechatqrcode)            | 100x100    | 7.04           | 37.68        | ---             | ---          | ---         |
+| [DaSiamRPN](./models/object_tracking_dasiamrpn)         | 1280x720   | 36.15          | 705.48       | 76.82           | ---          | ---         |
+| [YoutuReID](./models/person_reid_youtureid)             | 128x256    | 35.81          | 521.98       | 90.07           | 44.61        | ---         |
+| [MPPalmDet](./models/palm_detection_mediapipe)          | 256x256    | 15.57          | 89.41        | 50.64           | 145.56\*     | ---         |
```
Comment (fengyuentau) [inline README.md:36]: One PR for one purpose. Please create another PR for this change.

File: models/text_recognition_crnn/README.md [hunk 3]
```diff
@@ -38,10 +42,17 @@ python demo.py --input /path/to/image --model text_recognition_CRNN_CN_2021nov.o
 
 All files in this directory are licensed under [Apache 2.0 License](./LICENSE).
 
+## Examples
```
Comment (fengyuentau) [inline models/text_recognition_crnn/README.md:45]: The 'Examples' section should be put in `Demo` section under python commands and formatted as level-3 heading (`### Examples`).

**Review by fengyuentau — COMMENTED** [review]
Please see comments below.

**Comment by zihaomu** [comment]
@Charles-258, friendly reminder.

**Comment by Charles-258** [comment]
> @Charles-258, friendly reminder.

please review

**Review by fengyuentau — APPROVED** [review]
👍

**Pull request merged**